apps/web/src/pages/ManagerDashboardPage/ManagerDashboardPage.tsx

```diff
@@ -44,9 +45,15 @@ interface ManagerDashboardProps {
   userQuery: FragmentType<typeof ManagerDashboardUser_Fragment>;
 }
 
+type AccordionItems = ("your_talent_searches" | "")[];
+
 const ManagerDashboard = ({ userQuery }: ManagerDashboardProps) => {
   const paths = useRoutes();
   const intl = useIntl();
+  const [accordionItems, setAccordionItems] = useLocalStorage(
+    "accordion_items",
```

Comment:
I think the key name of this accordion on this page is too generic since the storage is scoped for the entire origin.  
![image](https://github.com/user-attachments/assets/735594f0-90b7-45eb-b87b-1f7ff9464f5b)
I wouldn't be surprised if we want to start storing other accordions on other pages, too.

You can see graphiql is using colons.  We could do that or periods.  :thinking:  We could do something like page:widget:function.  For example managerDashboardPage:managerToolsAccordion:expandedItems.

Reply:
Another approach is to store a load a single state object for a page.  Something like:

Reply:
I changed it to your first suggestion. Very good catch. I think we use that accordion key on another page :laughing: . For the second suggestion, we could switch to that in the future if there are more states to store. Changed in [6151cf9](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11579/commits/6151cf91eda6fd317e4591f8bf6a7bf5b9048b2f)